esy/bin/esyCommand.ml

```diff
@@ -431,6 +431,16 @@ let makeExecCommand
     else return ()
   in
 
+  let%bind command = RunAsync.ofRun (
+    let open Run.Syntax in
+    let%bind script = SandboxTools.findScript ~sandbox:info.sandbox command in
+    let command = script |> function
```

Comment:
Why `|>` instead of `match script with`? It seems to be doing exactly the same thing.